src/utilities.ts

```diff
@@ -229,3 +225,15 @@ export async function QueuePush(user: BlockUser): Promise<void> {
 		'unable to push user to queue',
 	);
 }
+
+export function UsernameElement(userName: string, screenName: string): HTMLParagraphElement {
+	const p = document.createElement("p");
+	const a = document.createElement("a");
+	a.href = "https://twitter.com/" + screenName;
```

Comment:
does this need to handle the URL change?

Reply:
maybe, but I don't really care tbh

Reply:
something to possibly address with a future PR then, I guess...